src/Toml/Type/Printer.hs

```diff
@@ -171,6 +175,19 @@ prettyKeyValue options i = mapOrdered (\kv -> [kvText kv]) options . HashMap.toL
     showText :: Show a => a -> Text
     showText = Text.pack . show
 
+    showTextUnicode :: Text -> Text
```

Comment:
I think, adding a few lines of documentation to this function would help to understand it better 🙂 Something like "this function converts `Text` to `String`, processes each character to decide how to pretty-print it and converts back to `Text`"